depends/tthread/tinythread.cpp

```diff
@@ -171,13 +171,17 @@ void * thread::wrapper_function(void * aArg)
   catch(...)
   {
     // Uncaught exceptions will terminate the application (default behavior
-    // according to the C++0x draft)
+    // according to C++11)
     std::terminate();
   }
 
+#if 0
```

Comment:
I might've missed a discussion about this, but is this something that should be fixed upstream, or is it DFHack-specific for some reason?